pkg/capabilities/cli/cmd/generate-types/main.go

```diff
@@ -18,6 +19,7 @@ var goWorkflowTestTemplate string
 
 var dir = flag.String("dir", "", fmt.Sprintf("Directory to search for %s files, if a file is provided, the directory it is in will be used", cmd.CapabilitySchemaFilePattern.String()))
 var localPrefix = flag.String("local_prefix", "github.com/smartcontractkit", "The local prefix to use when formatting go files")
+var extraUrls = flag.String("extra_urls", "", "Comma separated list of extra URLs to fetch schemas from")
```

Comment:
’m not thrilled about requiring the user to specify which remote files they’re referring to, but I needed to do so in order to construct the namespace correctly for go-jsonschema to work as intended with my generator.

I think it’s acceptable because it forces users to be more conscious of their inclusions, especially when nested. Users probably should know what they depend on for a capability...